_studio/shared/umc/codec/h264_dec/src/umc_h264_dec_bitstream_headers.cpp

```diff
@@ -1111,6 +1111,10 @@ Status H264HeadersBitstream::GetSequenceParamSetMvcExt(H264SeqParamSetMVCExtensi
 
         // decode the level's profile idc
         levelInfo.level_idc = (uint8_t) GetBits(8);
+        if (H264VideoDecoderParams::H264_LEVEL_MAX <= levelInfo.level_idc)
```

Comment:
According to our code:
```
_studio/shared/umc/codec/h264_dec/include/umc_h264_dec.h:87:        H264_LEVEL_MAX  = 52,
_studio/shared/umc/codec/h264_dec/include/umc_h264_dec.h:86:        H264_LEVEL_52   = 52,
```
Do I understand correctly that after this change all H264 level 5.2 will stop working and will return an error?

@dmitry-gurulev : we support levels up to 5.2, but not including 5.2?

Reply:
@dvrogozh, +1 we support 5.2, so proposed condition is incorrect.
@aulitin, I'd rather extract level check from _H264HeadersBitstream::GetSequenceParamSet_ into separate inline function, let's say `inline bool CheckLevel(uint8_t profile_idc, uint8_t level_idc)` and use it both at `GetSequenceParamSet` and `GetSequenceParamSetMvcExt`